site/src/components/keyboard-controls/KeyboardControls.module.css

```
@@ -3,14 +3,23 @@
   margin-bottom: calc(2 * var(--salt-size-unit));
 }
 
-.table td {
-  vertical-align: top;
-}
-
 .table td > :last-child {
   margin-bottom: 0;
 }
 
 .keyCol {
   min-width: 10rem;
 }
+
+.key {
+  display: inline-block;
+  background-color: #ffffff;
+  border: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-primary-borderColor);
+  border-radius: 3px;
+  font-family: var(--salt-text-label-fontFamily);
+  font-size: var(--salt-text-label-fontSize);
+  font-weight: var(--salt-text-fontWeight-strong);
+  margin: var(--salt-spacing-25);
+  padding: var(--salt-spacing-25) var(--salt-spacing-75);
+  box-shadow: var(--salt-color-gray-40) 0px 2px;
```

As @origami-z said we should use a shadow token here have you tried:

```suggestion
  box-shadow: var(--salt-overlayable-shadow);
```

---

I don't think this looks right, but I've updated the colour so it changes with theme